### PR TITLE
doc/website/blog: gormviz blog

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

**Review by a8m — APPROVED** [review]

File: doc/website/blog/2023-08-08-gorm-visualizations.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  (we'll see an example in a minute)
+* [Schema Loaders](https://atlasgo.io/docs/schema-loaders) - a feature in Atlas that allows users to load their
+  database schemas from different sources.
+
+In this blog post, we will create a toy GORM application, and show how you can use Atlas to visualize
+the database schema that GORM generates for you.
+
+### Step 1: Bootstrap the GORM application
+
+Start by creating a new directory for our project, and initialize a new Go module:
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-08-gorm-visualizations.md:25]:
``` suggestion
```
project. Then,

File: doc/website/blog/2023-08-08-gorm-visualizations.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  gorm.Model
+  Name   string
+  User   User
+  UserID uint
+}
+```
+
+### Step 2: Setup Atlas
+
+Now that we have a GORM application, let's setup Atlas. First, we will install the Atlas CLI:
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-08-gorm-visualizations.md:63]:
```suggestion
```
set up (two words when it's a verb, one word "setup" when a noun)

File: doc/website/blog/2023-08-08-gorm-visualizations.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+```go title="tools.go"
+//go:build tools
+package main
+
+import _ "ariga.io/atlas-provider-gorm/gormschema"
+```
+Alternatively, you can simply add a blank import to the `models.go` file we created
+above.
+
+Finally, to tidy things up run:
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-08-gorm-visualizations.md:89]:
```suggestion
```
run,

File: doc/website/blog/2023-08-08-gorm-visualizations.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+```
+This file defines two things:
+* An [External Schema Loader](https://atlasgo.io/atlas-schema/projects#data-source-external_schema) named `gorm` that will load our GORM models
+  into Atlas by executing the `ariga.io/atlas-provider-gorm` program on our `models` package. 
+* An [Environment](/atlas-schema/projects#environments) named `gorm` that will use the `gorm` schema loader
+  to load the models into a SQLite database in memory.
+
+### Step 3: Visualize the GORM models
+
+Now that we have everything setup, let's run Atlas and see what we get:
```
Comment (gedalyah-ariga) [inline doc/website/blog/2023-08-08-gorm-visualizations.md:125]:
```suggestion
```
set up (2 words)

**Pull request merged**